### PR TITLE
FIX: Add httpOnly true config to session cookie

### PR DESCRIPTION
**Pull request by tomxelliott**

### CODE REVIEW

File: src/main/modules/session/index.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export class SessionStorage {
         saveUninitialized: false,
         secret: config.get('session.secret'),
         cookie: {
-          httpOnly: false,
+          httpOnly: true,
```
Comment (rajaram-nagarajan) [inline src/main/modules/session/index.ts:25]: @tomxelliott - Looks like the respective test file needs to be fixed too

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by Shailesh-walke — DISMISSED** [review]

File: src/main/modules/session/index.ts (hunk 1 above)
Comment (tomxelliott) [reply src/main/modules/session/index.ts:25]: Yep, am on it

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**